mailbag/__init__.py

```diff
@@ -16,6 +16,25 @@
 mailbagit_args = bagit_parser.add_argument_group("Mailbag")
 # add mailbag-specific args here
 mailbagit_args.add_argument("--foo", help="The foo argument, you know, that one")
+mailbagit_args.add_argument("--i","--input_type", help="input type MBOX/PST/IMAP/EML/PDF/MSG",nargs=1)
+mailbagit_args.add_argument("--input_path", help="input path MBOX/PST/IMAP/EML/PDF",nargs=1)
+mailbagit_args.add_argument("--o","--output", help="output type MBOX/PST/EML/PDF/WARC",nargs=1)
```

Comment:
`--output` is the one that will need to allow for multiple args, as users can want to include multiple derivatives, such as `--output eml pdf warc`. Thinking about this more, it might be helpful to rename this `-d, --derivatives` as its not a mirror of the input type arg.